include/deal.II/dofs/dof_handler.h

```diff
@@ -1224,6 +1241,31 @@ DoFHandler<dim,spacedim>::get_fe () const
 
 template <int dim, int spacedim>
 inline
+const FiniteElement<dim,spacedim> &
+DoFHandler<dim,spacedim>::get_finite_element
+(const unsigned int) const
```

Comment:
should not we add an assert that `active_fe_index==0` instead of neglecting its value altogether?